avocado/core/requirements/resolver.py

```diff
@@ -0,0 +1,102 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2020
+# Authors: Willian Rampazzo <willianr@redhat.com>
+
+"""
+Requirements Resolver module.
+"""
+
+from ..nrunner import Runnable, Task, RUNNERS_REGISTRY_PYTHON_CLASS
+
+SUPPORTED_REQUIREMENTS = {}
+
+
+def create_requirements_tasks(tasks):
+    """Entry point for the Requirements Resolver.
+
+    This function isolates the Requirements Resolver object into this module.
+
+    :param tasks: List of tasks
+    :type tasks: list
+    """
+    resolver = RequirementsResolver(tasks)
+    return resolver.create_requirements_tasks()
+
+class UnsupportedRequirementType(Exception):
+    pass
+
```

Comment:
Style hint, this file doesn't have two blank lines between classes.  [PEP 8](https://www.python.org/dev/peps/pep-0008/#blank-lines)

Reply:
Opss, indeed, good catch!

Reply:
I'm wondering why `inspekt-style.sh` did not get it.

Reply:
That's true I will investigate that